src/vm/i386/excepx86.cpp

```diff
@@ -2762,6 +2763,10 @@ StackWalkAction COMPlusThrowCallback(       // SWA value
     if (fGiveDebuggerAndProfilerNotification)
         EEToProfilerExceptionInterfaceWrapper::ExceptionSearchFunctionLeave(pFunc);
     return SWA_CONTINUE;
+#else  // FEATURE_PAL
```

Comment:
It would be better to just ifdef-out the blocks that call these methods so that we do not have to go back to fix this. Take a look at `FEATURE_PAL` ifdefs around GetCallerToken and GetImpersonationToken in other places.